sync/pacman-script

```diff
@@ -40,3 +41,18 @@ for codebase in $(yq eval -o=j $YAMLFILE | jq -cr '.[]'); do
     echo "Moving '${name}' from '${temporaryExtraction}' to '${targetDestination}'";
     cp -fa "${temporaryExtraction}" "${targetDestination}"
 done
+
+# remove any not needed files
+for codebase in $(yq eval -o=j $YAMLFILE | jq -cr '.[]'); do
+    name=$(echo $codebase | jq -r '.name' -)
+    destination=$(echo $codebase | jq -r '.destination' -)
+    remove=$(echo $codebase | jq -r '.remove' -)
+    if [ "$remove" = "null" ]; then
+        continue
+    fi
+    echo $remove | jq -c -r '.[]' | while read removeentry; do
+        toremove="$(realpath -m ${REPO_DIR}${destination}/${removeentry})"
+        echo "Removing '${toremove}'"
```

Comment:
This is quite verbose. Perhaps just print out "Removing files from ${name}" before the loop would be better?

Reply:
Can do, but losts of other parts of the script are verbose and I guess this is mostly on purpose to make debugging easier.
We could always wrap all of this echoing in a flag though to hide it during normal execution?